tests/attested_tls/oe_enclave/enc/private.pem

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Do you need this file?

Reply:
this is for the "oesign", I think we need to keep it here. But please correct me if I understand incorrectly.

Reply:
Verified with Xuejun, this file is needed for a known MRSIGNER on sgx-lkl enclave side.